test/collection-browser.test.ts

```diff
@@ -38,6 +56,9 @@ describe('Collection Browser', () => {
     await el.updateComplete;
 
     expect(searchService.searchParams?.query).to.equal('collection:foo');
+    expect(
+      el.shadowRoot?.querySelector('#big-results-label')?.textContent
+    ).to.contains('Results');
```

Comment:
Is there a corresponding test for the single 'Result' case?

Reply:
+1